docs/use cases/README.md

```diff
@@ -1,149 +1,172 @@
-        <font size=24 color=black>Package: <b>UCD_3.0
     end header
 
-    title
-        <font size=18 color=black>UC_8. Редагувати конфігурацію порталу
-        <font size=16 color=black>Діаграма прецедентів
-    end title
-
-
-    actor "Користувач" as User #eeeeaa
-    
-    package UCD_1{
-        usecase "<b>UC_1</b>\nПереглянути список \nзвітів" as UC_1 #aaeeaa
-    }
-    
-    usecase "<b>UC_1.1</b>\nЗастосувати фільтр" as UC_1.1
-    usecase "<b>UC_1.2</b>\nПереглянути метадані \nзвіту" as UC_1.2  
-    usecase "<b>UC_1.2.1</b>\nДати оцінку звіту" as UC_1.2.1  
-    usecase "<b>UC_1.2.2</b>\nПереглянути інформацію \nпро авторів звіту" as UC_1.2.2
-    
-    package UCD_1 {
-        usecase "<b>UC_4</b>\nВикликати звіт" as UC_4 #aaeeaa
-    }
-    
-    usecase "<b>UC_1.1.1</b>\n Використати \nпошукові теги" as UC_1.1.1  
-    usecase "<b>UC_1.1.2</b>\n Використати \nрядок пошуку" as UC_1.1.2
-    usecase "<b>UC_1.1.3</b>\n Використати \nавторів" as UC_1.1.3  
-    
-    
-    
-    User -> UC_1
-    UC_1.1 .u.> UC_1 :extends
-    UC_1.2 .u.> UC_1 :extends
-    UC_4 .d.> UC_1.2 :extends
-    UC_1.2 .> UC_1.2 :extends
-    UC_1.2.1 .u.> UC_1.2 :extends
-    UC_1.2.2 .u.> UC_1.2 :extends
-    UC_1 ..> UC_1.2.2 :extends
-    
-    
-    UC_1.1.1 -u-|> UC_1.1
-    UC_1.1.2 -u-|> UC_1.1
-    UC_1.1.3 -u-|> UC_1.1
-    
-    right footer
-        Аналітичний портал. Модель прецедентів.
-        НТУУ КПІ ім.І.Сікорського
-        Киів-2020
-    end footer
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
+
+        :Користувач відкриває вікно
+        з усіма завданнями;
+
+        :Користувач обирає потрібне
+        завдання;
+
+        :Користувач натискає
+        кнопку "Змінити дедлайн";
+        
+        :Користувач обирає дедлайн
+        для завдання;
+        
+        :Користувач зберігає
+        введену інформацію;
+
+    |Система|
+        :Система перевіряє
+        правильність надісланих даних;
+        note right #ffaaaa
+        <b>Можлива</b>
+        <b>LEAD_EX_02.01</b>
+        end note
+
+        :Система записує зміни;
+
+    |Користувач|
+        :Користувач завершує
+        взаємодію;
+        stop;
 
 @enduml
+```
 
-**Діаграма прецедентів**
+- ID: LEAD_03
+
+```md
+
+@startuml
+
+    left header
+
+        <font color=000 size=18><b>ID:</b> LEAD_03
+
+        <font color=000 size=16><b>НАЗВА:</b> Призначити завдання розробникам
+        <font color=000 size=16><b>УЧАСНИКИ:</b> Тімлід, Система
+
+        <font color=000 size=16><b>ПЕРЕДУМОВИ:</b> Наявність створених завдань
+
+        <font color=000 size=16><b>РЕЗУЛЬТАТ:</b> Призначене завдання розробникам
+
+        <font color=000 size=16><b>ВИКЛЮЧНІ СИТУАЦІЇ:</b> Відсутні
+
+        <font color=000 size=16><b>ОСНОВНИЙ СЦЕНАРІЙ:</b>
+
+    end header
 
-</center>
+    |Користувач|
+        start
+        :Користувач авторизується
+        в системі;
+
+        :Користувач відкриває вікно
+        з усіма завданнями;
```

Comment:
Would be better kinda:

1. User clicks the button "Tasks"
2. The system is sending task list to user

...